sources/packages/backend/apps/db-migrations/src/sql/NotificationMessages/Insert-offering-change-request-inprogress-with-student-message.sql

```diff
@@ -0,0 +1,8 @@
+INSERT INTO
+  sims.notification_messages(id, description, template_id)
+VALUES
+  (
+    13,
+    'Offering Change Request Inprogress With Student',
```

Comment:
It would be good to have the notification details also in the ticket.

Reply:
Notification details for offering change request In progress with student are present in the ticket. Not sure if Jason wants to change it to have a common template for all 3 scenarios - in progress with student, approved and declined by SABC.

Reply:
> It would be good to have the notification details also in the ticket.

By details I meant that the 'Offering Change Request Inprogress With Student' would be part of the ticket AC also.

Reply:
I will check it with @JasonCTang